src/docs/overview/concepts.md

```diff
@@ -58,70 +58,44 @@ through a job scheduler using a [fair-share algorithm][url_fairshare].
     For more information about data risk classifications, see the [Information
     Security Risk Classification page][url_risk].
 
-## What's a cluster?
 
-A computing cluster is a federation of multiple compute nodes (independent
-computers), most commonly linked together through a high-performance
-interconnect network.
+## Investing into Sherlock
 
-What makes it a "super-computer" is the ability for a program to address
-resources (such as memory, CPU cores) located in different compute nodes,
-through the high-performance interconnect network.
-
-![overview](img/cluster_overview.png)
+For users who need more than casual access to a shared computing environment,
+Sherlock also offers Faculty members the possibility to invest into additional,
+dedicated computing resources.
 
-On a computing cluster, users typically connect to [login nodes][g_login],
-using a secure remote login protocol such as [SSH][g_ssh]. Unlike in
-traditional interactive environments, users then need to prepare [compute
-jobs][g_jobs] to submit to a [resource scheduler][g_scheduler]. Based on a set
-of rules and limits, the scheduler will then try to match the jobs' resource
-requirements with available resources such as [CPUs][g_cpu], memory or
-computing accelerators such as [GPUs][g_gpu]. It will then execute the user
-defined tasks on the selected resources, and generate output files in one of
-the different [storage locations][url_storage] available on the cluster, for
-the user to review and analyze.
+Unlike traditional clusters, Sherlock is a collaborative system where the
+majority of nodes are purchased and shared by the cluster users, known as
+*owners*. Choosing from a standard set of server configurations supported by
+SRCC staff (known as the Sherlock *catalog*), PIs can purchase their own
+servers to add to the cluster.
```

Comment:
```suggestion
majority of nodes are purchased and shared by the cluster users. When a user (typically a PI)
purchases one or more nodes, they become an *owner*. Owners can choose from a standard set of server configurations supported by SRCC staff (known as the Sherlock *catalog*) to add to the cluster.
```.

Reply:
I think the definition of "owner" isn't clear enough - the way it's currently written it is referencing generic cluster users.

Reply:
Agreed, thanks for the correction!